### PR TITLE
Updates Splash Screen to Fall Theme

### PR DESCRIPTION
**Pull request by redreceipt**
Fixes: [#4460](https://newspring.atlassian.net/browse/SYS-4460)

Demo

Applies new fall theme images for splash screen on iOS and Android.

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review function and form on Web
- [x] Review new test logic

### CODE REVIEW

File: android/app/src/main/java/host/exp/exponent/generated/DetachBuildConstants.java [hunk 1]
```diff
@@ -3,6 +3,6 @@
 // This file is auto-generated. Please don't rename!
 public class DetachBuildConstants {
 
-  public static final String DEVELOPMENT_URL = "expba3ad9e96a6043fc95726c73cb381a27://t3-pit.newspring-builds.apollos.exp.direct:80";
+  public static final String DEVELOPMENT_URL = "expba3ad9e96a6043fc95726c73cb381a27://10.0.200.139:19000";
```
Comment (richarddubay) [inline android/app/src/main/java/host/exp/exponent/generated/DetachBuildConstants.java]: This should be the `t3` link, not the `10.` IP address

Comment (redreceipt) [reply android/app/src/main/java/host/exp/exponent/generated/DetachBuildConstants.java]: @IsaacHardy is this a line we are going to have to manually change with every new feature?

Comment (redreceipt) [reply android/app/src/main/java/host/exp/exponent/generated/DetachBuildConstants.java]: @IsaacHardy and where did the 'exp' line come from?

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Sat Sep 16 17:38:34 PDT 2017
+#Wed Sep 05 15:26:06 EDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```
Comment (richarddubay) [inline android/gradle/wrapper/gradle-wrapper.properties]: @IsaacHardy @redreceipt I don't believe this should be changed from 3.3, should it?

File: android/app/src/main/java/host/exp/exponent/generated/DetachBuildConstants.java (hunk 1 above)
Comment (richarddubay) [reply android/app/src/main/java/host/exp/exponent/generated/DetachBuildConstants.java]: We still need this changed back @redreceipt.

File: android/gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (redreceipt) [reply android/gradle/wrapper/gradle-wrapper.properties]: @IsaacHardy ???

Comment (IsaacHardy) [reply android/gradle/wrapper/gradle-wrapper.properties]: Yes. Do not change this.

Comment (redreceipt) [reply android/gradle/wrapper/gradle-wrapper.properties]: @IsaacHardy I switched back to 3.3 and it broke Android Studio.

screen shot 2018-09-20 at 11 58 37 am

Comment (redreceipt) [reply android/gradle/wrapper/gradle-wrapper.properties]: It builds fine on 4.4. Is there an AppCenter reason why it should be 3.3?

**Comment by redreceipt** [comment]
ok I think we're good now

**Comment by IsaacHardy** [comment]
@richarddubay What is the update on this? Can we get this in by EOD?

**Comment by richarddubay** [comment]
@IsaacHardy I can't test it. My Android Studio keeps telling me that it can't find the SDK even though I've done everything it's told me to do in order to find it. So I have to figure that out first. This is part of what made me so frustrated yesterday. It should just work ... but it doesn't.

**Comment by IsaacHardy** [comment]
@richarddubay Need me to help you with that?

**Pull request merged**